cms/static/cms/sass/components/_pagetree.scss

```diff
@@ -539,33 +520,33 @@
         display: block;
         height: $pagetree-lang-size;
     }
-}
-.cms-tree-lang-container span {
-    box-sizing: border-box;
-    display: inline-block;
-    width: $pagetree-lang-size;
-    height: $pagetree-lang-size;
-    border-radius: 100%;
-    // will change according to states
-    border: 2px solid $white;
-    background: $white;
-    &.empty {
-        border-color: $gray-light;
-        background-color: $white;
-    }
-    &.unpublished,
-    &.unpublishedparent {
-        border-color: $gray-light;
-        background-color: $gray-light;
-    }
-    &.published {
-        border-color: $color-success;
-        background-color: $color-success;
-    }
-    &.dirty {
-        animation: pulsate 2.5s ease-out infinite;
-        border-color: $color-primary;
-        background-color: $color-primary;
+    .cms-pagetree-dropdown-trigger span {
+        box-sizing: border-box;
+        display: inline-block;
+        width: $pagetree-lang-size;
+        height: $pagetree-lang-size;
+        border-radius: 100%;
+        // will change according to states
+        border: 2px solid $white;
+        background: $white;
+        &.empty {
+            border-color: $gray-light;
+            background-color: $white;
+        }
+        &.unpublished,
+        &.unpublishedparent {
+            border-color: $gray-light;
+            background-color: $gray-light;
+        }
+        &.published {
+            border-color: $color-success;
+            background-color: $color-success;
+        }
+        &.dirty {
```

Comment:
good catch, but will have to do that in another pr (this requires lots of changes all over the place)
it's mostly reindent